test/utils.py

```diff
@@ -615,3 +617,86 @@ def wait_pid_disappear(file, target_pid):
                 found = False
                 break
     assert not found
+
+
+def copy_data(dst, file_paths):
+    for path in file_paths:
+        shutil.copy(path, dst)
+
+
+def get_version(tt_cmd, tmpdir):
```

Comment:
Please move all `tt_cmd` related functions to the `tt_helper.py` file. They are not general utilities on in particular.

Reply:
Done.